src/controllers/api/updateQuestController.ts

```
@@ -4,7 +4,7 @@
 import { getJSONfromString } from "@/src/helpers/stringHelpers";
 import { updateQuestKey, IUpdateQuestRequest } from "@/src/services/questService";
 import { getQuestCompletionItems } from "@/src/services/itemDataService";
-import { addItem, combineInventoryChanges, getInventory } from "@/src/services/inventoryService";
+import { addItem, addItems, combineInventoryChanges, getInventory } from "@/src/services/inventoryService";
```

ESLint

[error] 'addItem' is defined but never used.

(@typescript-eslint/no-unused-vars)